api/tests/Feature/Auth/RegistrationTest.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+namespace Tests\Feature\Auth;
+
+use App\Entity\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Str;
+use Illuminate\Support\Facades\Hash;
+use Tests\TestCase;
+use Faker\Factory as Faker;
+
+class RegistrationTest extends TestCase
+{
+    use RefreshDatabase;
+
+    private $faker;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->faker = Faker::create();
+    }
+
+    public function test_registration_with_valid_data()
+    {
+        $user = [
+            'email' => $this->faker->email,
+            'name' => $this->faker->name,
+            'password' => '$2y$10$92IXUNpkjO0rOQ5byMi.Ye4oKoEa3Ro9llC/.og/at2.uheWG/igi',
+            'password_confirmation' => '$2y$10$92IXUNpkjO0rOQ5byMi.Ye4oKoEa3Ro9llC/.og/at2.uheWG/igi',
```

Comment:
Here should be a real password instead of hash (ex. `12345678`). Please apply this to all tests